app/assets/javascripts/discourse/lib/a11y-dialog-wrapper/addon/templates/components/a11y-dialog-wrapper.hbs

```diff
@@ -0,0 +1,27 @@
+<div id="a11y-dialog" class="a11y-dialog-container" aria-labelledby={{this.dialog.titleElementId}} aria-hidden="true">
```

Comment:
Do we want to surface the internal implementation:
- in the component name?
- in the css classes?

Reply:
We certainly don't have to... `a11y-` is not a particularly nice prefix. I'm happy to go with simpler naming, how about: 

- dialog-holder for the component name and element id (we need an ID)
- `dialog-` as prefix for the classes

Reply:
Dont have strong opinions here, anything with dialog sounds fine

Reply:
Cool, I went with the above.